File: templates/vibekit/squads/agent-forge/templates/agent-package/adapters/go/agent.go
// GENERATED in Fase 5 by packager. Go runtime adapter for this Agent Package.
// Implements the common AgentRuntime interface; reads ../../manifest.yaml as the
// source of truth. Switching provider = editing the manifest, not this file.
package agent

import "errors"

// Runtime is the common interface every adapter implements (invoke / invokeStream /
// preflight / estimate / onEvent). See the package README for the full contract.

// CreateAgent builds the runtime from the package manifest.
func CreateAgent(manifestPath string) error {
	return errors.New("agent-forge: Go adapter is a Fase 5 stub — not yet generated")
}

File: templates/contextkit/squads/agent-forge/templates/agent-package/adapters/python/agent.py
"""GENERATED in Fase 2 by packager. Python runtime adapter for this Agent Package.

Implements the common AgentRuntime interface; reads ../../manifest.yaml as the source
of truth. Switching provider = editing the manifest, not this file.

    class AgentRuntime(Protocol):
        def invoke(self, input) -> AgentOutput: ...
        def invoke_stream(self, input) -> Iterable[AgentChunk]: ...
        def preflight(self) -> HealthReport: ...     # checks the fallback-chain providers
        def estimate(self, input) -> CostEstimate: ...
        def on_event(self, handler) -> Unsubscribe: ...  # audit events
"""


def create_agent(manifest_path: str = "../../manifest.yaml", credentials: dict | None = None):
    raise NotImplementedError("agent-forge: Python adapter is a Fase 2 stub — not yet generated.")
